lib/providers/download/Nyaa/special.json

```diff
@@ -387,5 +387,32 @@
 	"Ushinawareta Mirai wo Motomete",
 
 	"Wish Upon the Pleiades":
-	"Houkago no Pleiades"
+	"Houkago no Pleiades",
+
+	"Kochin Pa!":
+	"Kochinpa!",
+
+	"JoJo's Bizarre Adventure (2012)":
+	"JoJo's Bizarre Adventure",
+
+	"Black Lagoon: The Second Barrage":
+	"Black Lagoon",
+
+	"Classroom☆Crisis":
+	"Classroom Crisis",
+
+	"Peeping Life TV: Season 1??":
+	"Peeping Life S1",
+
+	"Miss Monochrome: The Animation 2":
+	"Miss Monochrome S2",
+
+	"Miss Monochrome: The Animation 3":
+	"Miss Monochrome S3",
+
+	"Ore ga Ojou-sama Gakkou ni "Shomin Sample" Toshite Gets♥Sareta Ken":
```

Comment:
Escape the double quotes 

i.e `"Ore ga Ojou-sama Gakkou ni \"Shomin Sample\" Toshite Gets♥Sareta Ken"`